products/compute/api.yaml

```diff
@@ -8894,6 +8965,205 @@ objects:
+          to the BackendService.
+        required: true
+        update_verb: :POST
+        update_url: 'projects/{{project}}/regions/{{region}}/targetHttpProxies/{{name}}/setUrlMap'
+  - !ruby/object:Api::Resource
+    name: 'RegionTargetHttpsProxy'
+    kind: 'compute#targetHttpsProxy'
+    base_url: projects/{{project}}/regions/{{region}}/targetHttpsProxies
+    collection_url_key: 'items'
+    has_self_link: true
+    input: true
+    description: |
+      Represents a RegionTargetHttpsProxy resource, which is used by one or more
+      forwarding rules to route incoming HTTPS requests to a URL map.
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation': 'https://cloud.google.com/compute/docs/load-balancing/http/target-proxies'
+      api: 'https://cloud.google.com/compute/docs/reference/rest/beta/regionTargetHttpsProxies'
+    async: !ruby/object:Api::Async
+      operation: !ruby/object:Api::Async::Operation
+        kind: 'compute#operation'
+        path: 'name'
+        base_url: 'projects/{{project}}/regions/{{region}}/operations/{{op_id}}'
+        wait_ms: 1000
+      result: !ruby/object:Api::Async::Result
+        path: 'targetLink'
+      status: !ruby/object:Api::Async::Status
+        path: 'status'
+        complete: 'DONE'
+        allowed:
+          - 'PENDING'
+          - 'RUNNING'
+          - 'DONE'
+      error: !ruby/object:Api::Async::Error
+        path: 'error/errors'
+        message: 'message'
+    parameters:
+      - !ruby/object:Api::Type::ResourceRef
+        name: 'region'
+        resource: 'Region'
+        imports: 'name'
+        input: true
+        required: true
+        description: |
+          The region where the regional proxy resides.
+    properties:
+      - !ruby/object:Api::Type::Time
+        name: 'creationTimestamp'
+        description: 'Creation timestamp in RFC3339 text format.'
+        output: true
+      - !ruby/object:Api::Type::String
+        name: 'description'
+        description: 'An optional description of this resource.'
+        input: true
+      - !ruby/object:Api::Type::Integer
+        name: 'id'
+        description: 'The unique identifier for the resource.'
+        output: true
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        description: |
+          Name of the resource. Provided by the client when the resource is
+          created. The name must be 1-63 characters long, and comply with
+          RFC1035. Specifically, the name must be 1-63 characters long and match
+          the regular expression `[a-z]([-a-z0-9]*[a-z0-9])?` which means the
+          first character must be a lowercase letter, and all following
+          characters must be a dash, lowercase letter, or digit, except the last
+          character, which cannot be a dash.
+        input: true
+        required: true
+     # This field is present in the schema but as of 2019 Sep 23 attempting to set it fails with
```

Comment:
Is there a bug against the docs for this also? At the very least they can document it there in the meantime (you can do so by clicking on the "report error" button in the docs page)

Reply:
Will do.